modules/puppet_proxy/dependency_injection/dependencies.rb

```diff
@@ -1,45 +1,46 @@
-require 'puppet_proxy/runtime_configuration'
-
+# rubocop:disable Style/Lambda
```

Comment:
Ah yes, definitely disable it globally then as it's applicable to the whole codebase.